### DISCUSSION TITLE
Harvester network not show other harvester

### DISCUSSION THREAD

**Discussion by hedandan1989** [Chia-Network/chia-blockchain#2570] [General]
![ZQ$U`OHY088Y{KX1Y@YXS0T](https://user-images.githubusercontent.com/83118680/115955991-e25fb800-a52c-11eb-8f77-8958b647e69a.png)
![518FM``6TYQR)WU}SZHYYIP](https://user-images.githubusercontent.com/83118680/115955992-e4c21200-a52c-11eb-866e-9a0279a679dc.png)

In 'Last attempted proof' I can see my another machine's files.
but in 'Your Harvester Network' I can't see my harvester.
Why?
Harvester not using GUI,it just init -c with main manchine's ca and start harvester only.

**Comment by frangere** [comment]
Have same issue at here. Following doc at https://github.com/Chia-Network/chia-blockchain/wiki/Farming-on-many-machines to setup "Farming on many machines". From log, Harvester (without GUI) do receive signage from Full Node, and "Last Attempted Proof"  also show another harvester's plot, but "Your Harvester Network" does not list my harvester machine. Anyone can explain this issue?

**Reply by phantom-rabbit** [reply]
same

**Reply by joehuchina** [reply]
same .  very confused

**Reply by Falacio** [reply]
I have same issue

**Comment by frangere** [comment]
re-login by click Keys->Choose your key and then you can find your harvester machine at "Your Harvester Network". Right now, I can find my harvester machine.

**Reply by Falacio** [reply]
That worked for me too. Thanks

**Reply by ghost** [reply]
Thank you for you answer. It really helped me and I drove crazy before I found that topic :D

**Reply by WastidLeprechaun** [reply]
This worked for me too! Thanks!!!
